OCR/florence/api.py

```diff
@@ -33,9 +36,12 @@ async def embed():
     if not uploaded_file:
         return {"error": "File is missing"}, 400
 
+    file_path,file_type = uploaded_file.filename.split('.')
```

Comment:
will merge for now, but can you add some error handling here later in case someone gives some random files without extensions